maths/Boolean_expression.dart

```diff
@@ -0,0 +1,40 @@
+String evaluateBoolExpr(String s) {
```

Comment:
You could add some tests
```suggestion
import 'package:test/test.dart';

String evaluateBoolExpr(String s) {
```